Sources/AssistiveButton/AssistiveButton.swift

```diff
@@ -96,19 +109,42 @@ final class AssistiveButton: UIButton {
         tapHandler()
     }
 
+    @objc private func panGestureRecognized(_ recognizer: UIPanGestureRecognizer) {
+        let view = recognizer.view
+        switch recognizer.state {
+            case .began:
+                stopTimer()
+                beginPoint = recognizer.location(in: view)
+            case .changed:
+                recognizer.setTranslation(.zero, in: view)
+                let point = recognizer.location(in: view)
```

Comment:
Will `point` be equal to zero?

Reply:
I think thats could be zero in some cases but its ok 🤔